config/routes.rb

```diff
@@ -389,10 +389,10 @@
   get "r", to: redirect("users/rdvs", status: 301), as: "rdvs_short"
 
   # tkn est obligatoire pour s'assurer qu'il est possible de se connecter
-  get "r/:id/:tkn", to: (redirect do |path_params, req|
-    query_params = format_redirect_params(req.params)
-    "users/rdvs/#{path_params[:id]}#{query_params}"
-  end), as: "rdv_short"
+  get "r/:tkn" => "redirect#rdv_short_without_id", as: "rdv_short_no_id"
```

Comment:
super nitpick : on pourrait aligner le nom de la route et du path helper ? 


```suggestion
  get "r/:tkn" => "redirect#rdv_short_without_id", as: "rdv_short_without_id"
```

Reply:
J'ai l'impression que `rdv_short_no_id` sous-entend qu'il existe une version avec ID. Dans la perspective où l'on aimerait se débarrasser de la route avec ID d'ici quelques semaines, pourquoi ne pas dès maintenant appeler cette nouvelle route `rdv_short_from_token` ?

Reply:
Bien vu à vous deux

Effectivement je vais garder ta proposition François